index.html

```diff
@@ -0,0 +1,853 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/16d2250538.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>TNW</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo-flex flex">
+          <img
+            class="top-logo sec-1-img"
+            src="images/toplogo.png"
+            alt="Pic here"
+          />
+          <div class="float-left">
+            <div class="nav-1 flex">
+              <div class="nav-1-left">
+                <a href="#">News</a>
+                <a href="#">Events</a>
+                <a href="#">Business</a>
+                <a href="#">AMAs</a>
+                <a href="#">Space</a>
+                <a href="#">Terms & Conditions</a>
+              </div>
+
+              <div class="nav-1-right">
+                <a href="#">About</a>
+                <a href="#">Advertise</a>
+                <a href="#">Jobs</a>
+                <a href="#">Contact</a>
+              </div>
+            </div>
+            <div class="nav-2-3">
+              <div class="nav-2 flex">
+                <div class="nav-2-left">
+                  <a href="#">LATEST</a>
+                  <a href="#">HARD FORK</a>
+                  <a href="#">PLUGGED</a>
+                  <a href="#">README</a>
+                  <a href="#">GROWTH QUARTER</a>
+                  <a href="#" class="no-disp">SHIFT</a>
+                  <a href="#" class="no-disp">NEURAL</a>
+                </div>
+                <div class="nav-2-right">
+                  <i class="fas fa-search nav-2-items"></i>
+                  <i class="fa fa-bars hamberger" aria-hidden="true"></i>
+
+                  <img
+                    class="nav-2-items no-disp"
+                    src="images/tnw2020.png"
+                    alt="TNW 2020"
+                  />
+                  <span class="no-disp">34 :22 :28 :50 </span>
+                  <button class="nav-2-btn nav-2-items no-disp" type="button">
+                    GET TICKETS
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="nav-3 flex">
+          <a href="#"
+            >Join TNW,it is our 15th annual flagship event, online. Get tickets
+            <i class="fa fa-arrow-right" aria-hidden="true"></i
+          ></a>
+        </div>
+      </nav>
+    </header>
+    <section class="section-1 flex">
+      <div class="upper-flex-container">
+        <img class=" tnw2020-2" src="images/first-grid.png" alt="pic here" />
+        <img class="tnw2020-3" src="images/tnw2020-3.jpg" alt="pic here">
```

Comment:
- [ ] **OPTIONAL** That tnw2020-3 image is not required, it is not showing in the original, you can take it out